projects/rhai/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "rhai.rs"
+language: rust
+# TODO(silvergasp): Add primary contact
+primary_contact: ""
+main_repo: "https://github.com/rhaiscript/rhai.git"
+auto_ccs:
+  - nathaniel.brough@gmail.com
```

Comment:
> For fuzzing maybe you should make yourself as the primary contact?

I think I'll just leave you as the primary contact. I'll get CC'd in bug/build reports anyway if I leave my email in the auto_ccs field.